Objective-C/Tests/ReplicatorTest+Main.m

```diff
@@ -1763,6 +1772,76 @@ - (void) testPushAndPullExpiredDocument_SG {
     AssertNil([savedDoc stringForKey: propertyKey]);
 }
 
+#pragma mark - SG (TLS with chain of certs)
+/**
+ - Use the `certs.pem` & `certs.key` in the `LiteCore/Replicator/tests/data/ssl_config.json`
+    - "SSLCert": "../../../../../Objective-C/Tests/Support/chain-certs/certs.pem",
+    - "SSLKey":  "../../../../../Objective-C/Tests/Support/chain-certs/certs.key",
+ */
```

Comment:
I have updated LiteCore's test to use the cert chain. The new files with be `sg_cert.pem`, `sg_key.pem`, and `cert.pem` (only leaf).

Reply:
https://github.com/couchbase/couchbase-lite-core/commit/a6401bdcf3580fd71699e4e71b0c8b5abc309942